app/src/main/java/org/openobservatory/ooniprobe/domain/GetTestSuite.java

```diff
@@ -53,4 +53,28 @@ public AbstractSuite getFrom(Result result) {
 
         return testSuite;
     }
+
+    public AbstractSuite getFrom(Result result, List<String> inputs) {
```

Comment:
Because this method is only mean to rerun Web Connectivity results, I think we should give this function a more explanatory name. Additionally, we should probably define what should be the expected behavior if feeding this method with a result different from Web Connectivity, and write a test to ensure this is the case.